us_web_design_standards.info

```diff
@@ -1 +1,66 @@
-name = U.S. Web Design Standards
+name        = US Web Design Standards
+description = Open source UI components and visual style guide to create consistency and beautiful user experiences across U.S. federal government websites.
+screenshot  = screenshot.png
```

Comment:
Feel free to borrow the placeholder `screenshot.png` from the 8.x branch if you'd like.